src/python/library/tritonclient/grpc/_infer_stream.py

```diff
@@ -57,18 +57,25 @@ def __init__(self, callback, verbose):
         self._verbose = verbose
         self._request_queue = queue.Queue()
         self._handler = None
+        self._cancelled = False
         self._active = True
+        self._response_iterator = None
 
     def __del__(self):
         self.close()
 
-    def close(self):
+    def close(self, cancel_requests=False):
         """Gracefully close underlying gRPC streams. Note that this call
         blocks till response of all currently enqueued requests are not
         received.
         """
+        if cancel_requests and self._response_iterator:
```

Comment:
Update function docstring for close() that it won't block if cancel_requests is True